substrate/frame/support/src/lib.rs

```diff
@@ -942,6 +942,16 @@ pub mod pallet_prelude {
 	/// verbatim to the combined impl.
 	#[doc = docify::embed!("src/tests/inject_runtime_type.rs", derive_impl_works_with_no_aggregated_types)]
 	pub use frame_support_procedural::inject_runtime_type;
+	/// Provides a mechanism to register a default implementation for a given trait. This can
+	/// then be used in tandem with `[`#[derive_impl(..)]`](`frame_support::derive_impl`)` to
```

Comment:
```suggestion
	/// then be used in tandem with [`#[derive_impl(..)]`](frame_support::derive_impl) to
```

This is not valid syntax.